docs/guides/src/operation/table.md

```diff
@@ -0,0 +1,62 @@
+# Table Operation
+
+## Query Table Information
+Query table information via table_id
```

Comment:
Add a chapter to describe `system table`.

Reply:
I split Operation doc into 3 parts.